traits/traits.py

```diff
@@ -207,6 +207,75 @@ def date_editor ( ):
 
     return DateEditor
 
+def _is_hastraits_instance(handler):
```

Comment:
Is there a better choice of name here? `_is_hastraits_instance` seems misleading.

Reply:
`_expects_hastraits_instance`?